### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by tethryus**
Also updated powershell_script.sl with the prerequisites.

Signed-off by Sorin Moldovan: <sorin.moldovan@hpe.com>

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -12,39 +12,38 @@ beta content.
 The following is an overview of what is included in the ready-made content:
 
 + **cloudslang**
+  + **amazon:**
+    + **aws:** [Amazon Web Services (AWS)](https://aws.amazon.com/) is a collection of remote computing services that make up a cloud-computing platform. This folder contains content for starting, stopping and listing servers.
```
Comment (mihaitusa) [inline DOCS.md]: @tethryus  Maybe you can write a little more about it... the content within AWS now is ready to supply much more complex scenarios then start/stop/list instances WDYT?

File: content/io/cloudslang/base/powershell/powershell_script.sl [hunk 2]
```diff
@@ -9,6 +9,79 @@
...
+#!  Set-Item WSMan:\localhost\Service\AllowUnencrypted -value true
+#!  Set-Item WSMan:\localhost\Client\AllowUnencrypted -value true
+#!  Set-Item WSMan:\localhost\Client\TrustedHosts -value * -force
+#!  Set-Item WSMan:\localhost\Shell\MaxMemoryPerShellMB -value 3000
+#!  Set-Item WSMan:\localhost\Shell\MaxShellsPerUser -value 30
+#!  Set-Item WSMan:\localhost\Plugin\microsoft.powershell\Quotas\MaxMemoryPerShellMB -value 3000
+#!  Restart-Service winrm
+#!
+#!  Part 2. On PS host and client:
+#!  1b) Set-executionpolicy RemoteSigned on RAS and server
```
Comment (Bonczidai) [inline content/io/cloudslang/base/powershell/powershell_script.sl]: RAS may not be needed here, can we use a general term  for it? - multiple places

File: DOCS.md (hunk 1 above)
Comment (tethryus) [inline DOCS.md]: @mihaitusa I agree, just let me know what description would suffice, in order for anyone that would want to use the integration, to have a better understanding over all the capabilities it provides.

Comment (mihaitusa) [inline DOCS.md]: @tethryus Just a enumeration of areas covered will be enough. :smiley:

File: content/io/cloudslang/base/powershell/powershell_script.sl (hunk 2 above)
Comment (Bonczidai) [inline content/io/cloudslang/base/powershell/powershell_script.sl]: something non-OO term?

**Pull request merged**